### ISSUE TITLE
Pub/Sub regression in 0.7.0

### ISSUE THREAD

**Issue by ptinsley** [GoogleCloudPlatform/gcloud-ruby#605]
Prior to 0.7.0 messages came out as they went in, now I am seeing messages coming out as base64 encoded. Backing down to 0.6.2 or lower I get the expected result.

**Comment by quartzmo** [comment]
@ptinsley Thanks for reporting this! We are looking into it now.

**Comment by blowmage** [comment]
This is a regression. We'll get this fixed ASAP.

**Comment by ptinsley** [comment]
Awesome, thanks! Also potentially related, not sure if this would count as a regression but idle workers in a listen loop take a decent amount of time to honor an interrupt where previously it was almost instant. Is there a better way for me to signal workers to exit?

**Comment by quartzmo** [comment]
@ptinsley Are you still seeing that after reverting to <= 0.6.2?

**Comment by blowmage** [comment]
How long are you talking? 30 seconds? Longer?

**Comment by ptinsley** [comment]
Here are a couple of test kills on 0.7.0:


````
date && kill 24655 && fg || date
Fri Apr  1 14:58:36 CDT 2016
ruby pubsub-pull.rb
Terminated
Fri Apr  1 14:59:37 CDT 2016

date && kill 24617 && fg || date
Fri Apr  1 14:57:45 CDT 2016
ruby pubsub-pull.rb
Terminated
Fri Apr  1 14:57:51 CDT 2016
```

On 0.6.2 it's instant

**Comment by quartzmo** [comment]
@ptinsley Much thanks for promptly reporting these two issues! We've released a fix for the base64-encoding issue on [v0.7.1](https://github.com/GoogleCloudPlatform/gcloud-ruby/releases/tag/v0.7.1). Please verify that it works for you.

Unfortunately, the messages you created with 0.7.0 will remain base64-encoded in Pub/Sub.

**Comment by ptinsley** [comment]
Got some time to test and messages look good, thanks for the prompt response!
